wandb/cli/cli.py

```diff
@@ -1044,13 +1059,10 @@ def launch_sweep(
     if not args:
         return
 
-    # validate training job existence
-    if not sweep_utils.check_job_exists(PublicApi(), sweep_config.get("job")):
-        return False
-
-    # validate scheduler job existence, if present
-    if not sweep_utils.check_job_exists(PublicApi(), scheduler_job):
-        return False
+    # set name of scheduler
+    name = scheduler_args.get("name") or "scheduler.WANDB_SWEEP_ID"
```

Comment:
use the format the frontend uses, and pass it into the run setting object for consumption by the sweep scheduler.